website/highlights/2020-03-10-dedupe-transform.md

```diff
@@ -1,20 +1,42 @@
 ---
-last_modified_on: "2020-04-13"
+last_modified_on: "2020-04-16"
 $schema: "/.meta/.schemas/highlights.json"
 title: "New Dedupe Trasnform"
```

Comment:
```suggestion
title: "New Dedupe Transform"
```

Reply:
We should add a lint to detect spelling mistakes for commonly used terms in Vector 🥇

Reply:
Agree, opened https://github.com/timberio/vector/issues/2357